examples/sm/sm_main.cu

```diff
@@ -48,25 +48,30 @@ struct main_struct
 
         cudaError_t retval = cudaSuccess;
         util::CpuTimer cpu_timer;
-        GraphT graph;
+        GraphT data_graph;
+        GraphT pattern_graph;
 
         cpu_timer.Start();
-        GUARD_CU(graphio::LoadGraph(parameters, graph));
+        GUARD_CU(graphio::LoadGraph(parameters, data_graph));
+        GUARD_CU(graphio::LoadGraph(parameters, pattern_graph, "pattern-"));
         cpu_timer.Stop();
+        std::cout << "data_graph: " << data_graph.nodes << " " << data_graph.edges << std::endl;
+        std::cout << "query_graph: " << pattern_graph.nodes << " " << pattern_graph.edges <<std::endl;
```

Comment:
Can you please use `util::PrintMsg("data_graph: " + std::to_string(data_graph.nodes) ... , !quiet);` instead, so when someone uses the `quiet` flag, this isn't printed out?